### ISSUE TITLE
Maintainer gesucht

### ISSUE THREAD

**Issue by nerrixde** [nerrixDE/DSBApi#24]
Ich habe keinen DSB Zugang mehr und benutze die App und zugehörige Software nicht mehr.

Ich würde jemanden gerne Rechte in dieser Repo geben, der das ganze dann weiter führt. 

Dein GitHub Account sollte nicht 10 Minuten alt sein: Sprich das sollte nicht dein erstes Open Source Projekt sein, idealerweise. 

Gerne hier melden 👇🏻

**Comment by Neotastisch** [comment]
Hey, würde mich freuen es weiterzuführen, ist natürlich nicht mein erstes Projekt, habe über 40 Stück, meißt aber nicht Open-Source.

**Comment by nerrixDE** [comment]
@Neotastisch cool! Du kannst ja schonmal ein paar Pull Requests machen, dann werde ich dir auch Rechte in dieser Repo geben, dass du das selbst kannst

**Comment by Neotastisch** [comment]
Hey, sorry, verstehe nicht ganz was du meinst, soll ich ein Pull-Request machen um es zu verbessern, um es dir zu zeigen, oder was meinst du?

**Comment by nerrixDE** [comment]
genau, tue einfach so als wärst du schon maintainer. Nur halt anstatt direkt zu pushen machst du einen Pull Request und dann kann ich dir die Rechte geben

**Comment by Neotastisch** [comment]
Okey, bin dabei

**Comment by Neotastisch** [comment]
Muss aber zugeben, dass ich mir nicht ganz sicher bin was genau ich ändern kann, denn es funktioniert alles perfekt, ich werde anfangen die readme überarbeiten.

**Comment by nerrixDE** [comment]
jo

**Comment by asdat3** [comment]
Tut mir leid das so zu machen, die API geht bei mir aber einfach nicht bekomme wie andere auch ein leeres Array:
Dieses modul hingegen funktioniert Einwandfrei:
https://pypi.org/project/pydsb/

**Comment by neooriginal** [comment]
> Tut mir leid das so zu machen, die API geht bei mir aber einfach nicht bekomme wie andere auch ein leeres Array: Dieses modul hingegen funktioniert Einwandfrei: https://pypi.org/project/pydsb/

Hey, hatte die ReadMe mal überarbeitet, müsste so gehen

**Comment by BastelPichi** [comment]
Hai!
Könnte das übernehmen, habe auch schon ein anderes reverse-engineering projekt auf meinem account.
Könnte das später fertig machen.

Die API hat sich total verändert, der Code sollte um einiges kürzer werden, zumal viel unnötiger Code vorhanden ist. Würde versuchen downwoards compability nicht kaputtzumachen, obwohl dies an diesem punk wahrscheinlich auch keine rolle mehr spielt.
Viele Grüße

**Comment by nerrixde** [comment]
jo

**Comment by cybcon** [comment]
@nerrixde konntest du den pypi Account irgendwie fixen / zurücksetzen
Ohne den wird das Maintainen hier wenig Sinn machen.

**Comment by LennyMaxMine** [comment]
Würde das Projekt erstmal wieder zum funktionieren bringen, muss schauen ob ich das in Zukunft maintainen kann. Melde dich gerne wenn ich dir helfen könnte!

**Comment by Liese2007** [comment]
> Würde das Projekt erstmal wieder zum funktionieren bringen, muss schauen ob ich das in Zukunft maintainen kann. Melde dich gerne wenn ich dir helfen könnte!

@LennyMaxMine bist Du gerade dabei, das Projekt wiederzubeleben?
Klasse!

**Comment by LennyMaxMine** [comment]
Jo, nen funktionierender fork sollte die nächsten Tage rauskommen.

**Comment by LennyMaxMine** [comment]
@Liese2007 jo deine Mail ist angekommen, bei mir funktioniert diese Repo aktuell und habe gerade etwas viel zu tuen, weshalb das maintainen erstmal pausiert wurde. Wie davor schon erwähnt bräuchte ich allerdings auch zugriff auf den Pypi account. 
Vielleicht kannst du kurz sagen was bei dir nicht funktioniert, dann könnten wir an einer Lösung arbeiten.

**Comment by Liese2007** [comment]
Moin @LennyMaxMine ,
ich hatte noch gar kein Versuch gestartet, weil ich überall las, dass es Probleme gibt.
Will es in Home Assistant zum laufen bringen und Versuche es jetzt mal mit der Python Script Integration.

Bin allerdings kein Coder und das wäre das erste Python Script, dass ich nutze. 

Starte dann mal meine Odyssee:-)
Danke für die Infos!
